generate_figures/testing_utilities.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+import wandb
+from utilities import get_results_by_tag
+
+project = "highdimensionaleconlab/symmetry_dynamic_programming"
+
+#for one runs
+df_nu_150_test_results = get_results_by_tag(project,"deep_sets_nonlinear_nu_150_one_run", test_results=True)
+df_nu_150 = get_results_by_tag(project,"deep_sets_nonlinear_nu_150_one_run")
+
+#for multiple artifact ones I stopped it after grabbing 5
+baseline_deep_moments_test_results = get_results_by_tag(project,"baseline_deep_moments", test_results=True)
+baseline_deep_moments = get_results_by_tag(project,"baseline_deep_moments")
```

Comment:
These were just random ones I grabbed to see if it was working for one and then multiple. Also what is `hpo_objective` and why is it sometimes NaN.....is it because its set in the yaml as `hpo_objective_name: test_loss` and then when it fails its test loss the `hpo_objective` just becomes NaN?

Reply:
That sounds right to me.  Does it break anything?  If not I think we are good to go.

Reply:
@jlperla no it works I just dont like nans